parlai/crowdsourcing/tasks/wizard_of_internet/constants.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Possible roles of an agent during task
+NO_ROLE = 0
+WIZARD = 1
+APPRENTICE = 2
+IN_TRAINING = 10
+WIZARD_IN_TRAINING = WIZARD + IN_TRAINING
+APPRENTICE_IN_TRAINING = APPRENTICE + IN_TRAINING
+# The role_id to role_name mapping
+ROLE_NAMES = {WIZARD: 'Wizard', APPRENTICE: 'Apprentice'}
+
+# The keys to get agent qualification data from opt.
+SAVED_DATA_WORKER_KEY = 'worker'
+SAVED_DATA_IS_WIZARD_KEY = 'is_wizard'
+SAVED_DATA_ROLE_QUALIFICATION_DATA_KEY = 'qualification_dict'
+ROLE_QUALIFICATION_NAME_KEY = 'role_qname'
+
+# OnBoardingSteps
+# NOTE: Make sure these number are consistent with OnboardingSteps,
+# as they are defined in the SidePane.jsx frontend file.
+ONBOARDING_STEPS = {
+    'NOT_ONBOARDING': 0,
+    'CHAT_INTERFACE': 1,
+    'TRY_SEARCH': 2,
+    'PERSONA_WIZARD': 3,
+    'PERSONA_APPRENTICE': 4,
+    'WAITING': 10,
+}
+
+# Name of (bot)agents involved in the task world
+ONBOARDING_AGENT = 'OnboardingBot'
+PERSONA_AGENT = 'PersonaAgent'
+SEARCH_AGENT = 'SearchAgent'
+COORDINATOR_AGENT = 'Coordinator'
+
+# NOTE: do not forget to change ONBOARDING_PERSONA_KEYWORDS below if changing this
```

Comment:
could you maybe elaborate how/why to change?

Reply:
Done!